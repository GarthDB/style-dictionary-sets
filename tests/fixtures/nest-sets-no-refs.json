{
  "component": {
    "size": {
      "sets": {
        "subSystemOne": {
          "sets": {
            "mobile": {
              "value": "15px"
            },
            "desktop": {
              "value": "12px"
            }
          }
        },
        "subSystemTwo": {
          "sets": {
            "mobile": {
              "value": "16px"
            },
            "desktop": {
              "value": "13px"
            }
          }
        }
      }
    }
  }
}
