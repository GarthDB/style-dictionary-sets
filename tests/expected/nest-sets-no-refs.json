{
  "component": {
    "size": {
      "sets": {
        "spectrum": {
          "sets": {
            "mobile": {
              "value": "15px",
              "filePath": "tests/fixtures/nest-sets-no-refs.json",
              "isSource": true,
              "original": {
                "value": "15px"
              },
              "name": "component-size-sets-spectrum-sets-mobile",
              "attributes": {
                "sets": ["spectrum", "mobile"]
              },
              "path": [
                "component",
                "size",
                "sets",
                "spectrum",
                "sets",
                "mobile"
              ]
            },
            "desktop": {
              "value": "12px",
              "filePath": "tests/fixtures/nest-sets-no-refs.json",
              "isSource": true,
              "original": {
                "value": "12px"
              },
              "name": "component-size-sets-spectrum-sets-desktop",
              "attributes": {
                "sets": ["spectrum", "desktop"]
              },
              "path": [
                "component",
                "size",
                "sets",
                "spectrum",
                "sets",
                "desktop"
              ]
            }
          }
        },
        "express": {
          "sets": {
            "mobile": {
              "value": "16px",
              "filePath": "tests/fixtures/nest-sets-no-refs.json",
              "isSource": true,
              "original": {
                "value": "16px"
              },
              "name": "component-size-sets-express-sets-mobile",
              "attributes": {
                "sets": ["express", "mobile"]
              },
              "path": [
                "component",
                "size",
                "sets",
                "express",
                "sets",
                "mobile"
              ]
            },
            "desktop": {
              "value": "13px",
              "filePath": "tests/fixtures/nest-sets-no-refs.json",
              "isSource": true,
              "original": {
                "value": "13px"
              },
              "name": "component-size-sets-express-sets-desktop",
              "attributes": {
                "sets": ["express", "desktop"]
              },
              "path": [
                "component",
                "size",
                "sets",
                "express",
                "sets",
                "desktop"
              ]
            }
          }
        }
      }
    }
  }
}
